pkg/types/vulnerability.go

```diff
@@ -13,6 +13,7 @@ type DetectedVulnerability struct {
 	FixedVersion     string       `json:",omitempty"`
 	Layer            ftypes.Layer `json:",omitempty"`
 	SeveritySource   string       `json:",omitempty"`
+	URL              string       `json:",omitempty"`
```

Comment:
We already have `References`. Can we add an AVD link to the field?
https://github.com/aquasecurity/trivy-db/blob/master/pkg/types/types.go#L126

In that case, we may not need to generate AVD links in `convert.go` because a server calls `FillInfo` and `References` should have the AVD link. We can just filter and extract the AVD link from `References` in `writer.go`.